packages/util-dev/src/k6-utils.ts

```diff
@@ -0,0 +1,258 @@
+import { Cardano } from '@cardano-sdk/core';
+import http from 'k6/http';
```

Comment:
Could you please test if `web-extension` e2e tests still work after this change?

Reply:
If I'm not wrong @mirceahasegan , you did some researches about running this test in a headless mode... but some times have passed.
Could it be a good idea to do a new attempt?
Hopefully some required tools have been upgraded in the mean while removing the blocker you found on last attempt.

Reply:
I did some research, but hit a dead end due to the web extension part which does not play well in headless mode.
> Could it be a good idea to do a new attempt?
It would be nice, but it's quite low on the priority list.

Reply:
> Could you please test if web-extension e2e tests still work after this change?

They work.